public/scripts/client.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @returns {Object} - returns the content of data.json in a parsed form
+ */
+
+export async function fetchData() {
+    try {
+      const response = await fetch("http://localhost:3000/data");
+      const result = await response.json();
+      return result;
+    } catch (error) {
+      console.error(error);
+      return error.message;
+    }  
+}
+/** 
+* @param {Object} getUpdatedData - expects an object wich will !!REPLACE!! the data.json file content 
+*/
+
+export async function changeData(getUpdatedData) {
```

Comment:
The name “changeData” is pretty generic; couldn’t it be updateData instead? Maybe addData bc it is a POST?